### PR TITLE
Fix public immutable state

### PR DESCRIPTION
**Pull request by prateekdefi**
Issue: https://linear.app/perennial/issue/PE-1789/remove-public-states-set-in-constructor-from-upgradeable-contracts

### CODE REVIEW

**Review by EdNoepel — APPROVED** [review]
LGTM.  Happy to help merge this with PR #451 .

File: packages/perennial-account/contracts/Controller_Incentivized.sol [hunk 1]
```diff
@@ -36,30 +36,31 @@ abstract contract Controller_Incentivized is Controller, IRelayer, Kept {
     KeepConfig public keepConfig;
 
     /// @dev Handles relayed messages for nonce cancellation
-    IVerifierBase public nonceManager;
+    IVerifierBase public immutable nonceManager;
```
Comment (arjun-io) [inline packages/perennial-account/contracts/Controller_Incentivized.sol:39]: can we move this above the mutable keepConfig just to make the storage layout clearer?

**Review by arjun-io — COMMENTED** [review]
one nit but lgtm overall

File: packages/perennial-account/contracts/Controller_Incentivized.sol (hunk 1 above)
Comment (prateekdefi) [reply packages/perennial-account/contracts/Controller_Incentivized.sol:36]: Fixed in [7faab93](https://github.com/equilibria-xyz/perennial-v2/pull/449/commits/7faab93e5638913e31d0222e9b070ceb27a05592)

**Review by arjun-io — APPROVED** [review]

**Review by kbrizzle — APPROVED** [review]

**Pull request merged**